release-content/0.16/migration-guides/hash_explicit_types.md

```diff
@@ -0,0 +1,25 @@
+`hashbrown`'s default hasher [has changed in `hashbrown 0.15`](https://github.com/rust-lang/hashbrown/pull/563), resulting in serious regressions in type inference and method availability for Bevy users of `HashSet` and `HashMap`.
+We have updated versions to avoid compiling duplicate copies of the crate, as Bevy's own dependencies have already updated.
+
+Bevy uses a different (faster on average, but slower in the face of malicious inputs) than `hashbrown`'s new default hasher.
```

Comment:
Looks like a word or two is missing after different

Reply:
```suggestion
Bevy uses a different hasher (faster on average, but slower in the face of malicious inputs) than `hashbrown`'s new default hasher.
```